docs/src/use/core-concepts/glossary.md

```diff
@@ -0,0 +1,397 @@
+
+## L
+
+### Legacy Config
+
+The previous configuration file format for ESLint, now superseded by ["Flat" config](#flat-config).
+
+Legacy ELSint configurations are named in the format `.eslintrc.*` and allowed to be nested across files within sub-directories in a project.
+
+You can read more about the lifetime of legacy configurations in [ESLint's new config system, Part 1: Background](https://eslint.org/blog/2022/08/new-config-system-part-1).
+
+### Linter
+
+A [static analysis](#static-analysis) tool that can [report](#report) the results from running a set of [rules](#rule) on source code.
+Each rule may report any number of [violations](#violation) in the source code.
+
+ESLint is a commonly used linter for JavaScript and other web technologies.
+
+Note that a _linter_ is separate from [formatters](#formatter-tool) and [type checkers](#type-checker).
+
+### Logical (Rule)
+
+A [rule](#rule) that looks at logical issues in code.
+
+Many logical rules look for likely crashes (e.g. [`no-undef`](../../rules/no-undef)), unintended behavior (e.g. [`no-sparse-arrays`](../../rules/no-sparse-arrays)), and unused code (e.g [`no-unused-vars`](../../rules/no-unused-vars)),
+
+You can see the full list of logical rules that ship with ESLint under [Rules > Possible Problems](../../rules/#possible-problems)
+
+## N
+
+### Node
+
+A section of code within an [AST](#abstract-syntax-tree-ast).
+
+Each node has a node type representing its type of syntax.
+For example, the `1 + 2` in the AST for `1 + 2;` is a _BinaryExpression_.
+
+ESLint uses the [ESQuery](#esquery) format for its AST and [selectors](#selector).
+
+## O
+
+### Override
+
+When a [config file](#config-file-configuration-file) sets a new severity and/or rule options for a rule in a set of files.
+
+Overrides are supported in both configuration formats.
+
+The following [flat config](#flat-config) overrides `no-unused-expression` from `"error"` to `"off"` in `*.test.js` files:
+
+```js
+export default [
+  rules: {
+    "no-unused-expressions": "off"
+  },
+  {
+    files: ["*.test.js"],
+    rules: {
+        "no-unused-expressions": "off"
+    }
+  }
+];
+```
+
+...and the following [legacy config](#flat-config) also overrides `no-unused-expression` from `"error"` to `"off"` in `*.test.js` files:
+
+```js
+{
+  "rules": {
+    "no-unused-expressions": "off"
+  },
+  "overrides": [
+    {
+      "files": ["*.test.js"],
+      "rules": {
+        "no-unused-expressions": "off"
+      }
+    }
+  ]
+}
+```
+
+Overrides are much simpler in flat configs, as flat configs declare all rules in one file rather than allowing nested hierarchies of configs the way legacy configs do.
+
+For more information on overrides in legacy configs, see [Configuration Files (Deprecated) > How to overrides work?](../configure/configuration-files-deprecated#how-do-overrides-work).
+
+## P
+
+### Parser
+
+A package that reads in a string and converts it to an [AST](#abstract-syntax-tree-ast).
```

Comment:
```suggestion
An object that reads in a string and converts it to an [AST](#abstract-syntax-tree-ast).
```

Also, not sure how specific you were planning on getting here, but technically a parser in the outside world doesn't have to return an AST. If your intent is to be ESLint-specific, then I might instead say that a parser in ESLint takes in source code and returns an AST.

Reply:
Yes! +1 to mentioning it's not an ESLint-specific concept. Reworded to mention "a standardized format" and that ESLint specifically asks for an AST.